### DISCUSSION TITLE
Copilot autocomplete comments contain profanity

### DISCUSSION THREAD

**Discussion by EricWittrock** [community/community#6855] [Copilot Conversations]
Just a heads up, some of the autocomplete comments can contain profanity.
![image](https://user-images.githubusercontent.com/92955915/139155057-4fc25e7a-335e-4604-97b5-54190b503a59.png)
This obviously came up after a mistype when writing function, but I'm a bit concerned that these words can come up anywhere.

**Comment by Urethramancer** [comment]
It's probably trained on my public repos.
